src/reflect/scala/reflect/internal/Scopes.scala

```diff
@@ -274,13 +274,13 @@ trait Scopes extends api.Scopes { self: SymbolTable =>
 
     /** Returns an iterator yielding every symbol with given name in this scope.
      */
-    def lookupAll(name: Name): Iterator[Symbol] = new Iterator[Symbol] {
+    def lookupAll(name: Name): Iterator[Symbol] = new AbstractIterator[Symbol] {
       var e = lookupEntry(name)
```

Comment:
private[this] ?

Reply:
https://github.com/scala/scala/pull/7127